### PR TITLE
💄 Add padding to tasklist

### PR DESCRIPTION
**Pull request by LeoTT**
## What did you change?

**Changes:**
Adds padding to tasklist. 
screen shot 2018-08-07 at 09 47 12

Fixes: 
#692 
#799 

## How can others test the changes?

Open task list, look at layout (accessible via dashboard->`User tasks` of a running process).

## PR-Checklist

Please check the boxes in this list after submitting your PR:

- [ ] You can merge this PR **right now** (if not, please prefix the title with "WIP: ")
- [x] I've tested **all** changes included in this PR.
- [x] I've also reviewed this PR myself before submitting (e.g. for scrambled letters, typos, etc.)
- [ ] I've rebased the `develop` branch with my branch before finishing this PR.
- [x] I've **not added any other changes** than the ones described above.
- [x] I've mentioned all **PRs, which relate to this one**

### CODE REVIEW

**Comment by LeoTT** [comment]
The problem is that currently every subsite needs to implement its own container with padding.
If we wanted to keep that system we could do:
```css
.dashboard-container  .tasklist-container {
  padding: 0;
}
```

How about the following instead?
```css
.router-view {
  position: relative;
  padding: 60px 20px;  
}

.process-def-detail {
  position: absolute;
  top: 0;
  left: 0;
  height: 100%;
  width: 100%;
}
```
That allows us to remove a couple div containers and classes and makes composition easier.
`.process-def-detail` would have to have position absolute in order to circumvent the standard padding(negative margin also possible, but more irritating imo).

**Comment by Vyperus** [comment]
The subsite (like task-list)  should display itself in the space that it gets from the "container" of the site that is displaying it. It shouldn't have a padding for itself since the displaying site is repsonsible for positioning the elements.

**Comment by SteffenKn** [comment]
@Vyperus The problem here is that the task list is used in the dashboard but also as an own page. For example when display user task for a specific correlation. Then the task-list has 100% width which looks disadvantageous.

**Comment by cmg-dev** [comment]
Possible Solution:

Don't use the same list element twice!

Copy them and break the two lists apart.

File: src/modules/dashboard/dashboard.ts [hunk 1]
```diff
@@ -1,23 +1,47 @@
-import {ForbiddenError, isError, UnauthorizedError} from '@essential-projects/errors_ts';
-import {IManagementApiService, ManagementContext} from '@process-engine/management_api_contracts';
+import {
```
Comment (Vyperus) [inline src/modules/dashboard/dashboard.ts]: Sort imports after known schema.

1) Framework
2) Thirdparty
3) Intern

File: src/modules/dashboard/dashboard.ts [hunk 2]
```diff
@@ -95,4 +160,66 @@ export class Dashboard {
...
+    return userTasksAndProcessModels;
+  }
+
+  private async _updateUserTasks(): Promise<void> {
+    try {
+      this._userTasks = await this._getUserTasks();
+      this.succesfullRequested = true;
+    } catch (error) {
+      if (isError(error, UnauthorizedError)) {
+        this._notificationService.showNotification(NotificationType.ERROR, 'You don\'t have permission to view the task list.');
```
Comment (cmg-dev) [inline src/modules/dashboard/dashboard.ts]: > You do not have **the** permission (...)

File: src/modules/dashboard/dashboard.ts [hunk 3]
```diff
@@ -1,23 +1,49 @@
...
+  processModel: ProcessModelExecution.ProcessModel;
+}
+
 @inject('ManagementApiClientService', 'NotificationService', 'AuthenticationService', Router)
 export class Dashboard {
 
   public showTaskList: boolean = false;
   public showProcessList: boolean = false;
+  public currentPage: number = 0;
+  public pageSize: number = 10;
```
Comment (cmg-dev) [inline src/modules/dashboard/dashboard.ts]: We should put this into a dashboard config object.

This should also come from the environment of aurelia.

Please create an issue and todo with this.

File: src/modules/dashboard/dashboard.ts [hunk 4]
```diff
@@ -49,6 +75,45 @@ export class Dashboard {
     return true;
   }
 
+  public async activate(): Promise<void> {
+    this._getUserTasks = this._getAllUserTasks;
+    this._updateUserTasks();
+  }
+
+  public attached(): void {
+    this._getUserTasksIntervalId = window.setInterval(() => {
```
Comment (LeoTT) [inline src/modules/dashboard/dashboard.ts]: When is this interval cleared?

**Comment by Vyperus** [comment]
@cmg-dev Done!

**Pull request merged**